docs/getting-started/quick-start.md

```diff
@@ -70,8 +70,8 @@ class DefaultListComponent(
     componentContext: ComponentContext,
     private val onItemSelected: (item: String) -> Unit,
 ) : ListComponent {
-    override val model: Value<Model> =
-        MutableValue(Model(items = List(100) { "Item $it" }))
+    override val model: Value<ListComponent.Model> =
```

Comment:
In my editor this requires referencing the interface to access nested data class. As I have longer component names this becomes quite long line.

override val model: Value<RecipeListScreenComponent.Model> = MutableValue(RecipeListScreenComponent.Model(emptyList()))

What I did was to extract model class outside the interface so it looks a bit nicer to me.

Reply:
It's up to you, but in the documentation and samples I prefer to keep the Model class inside the component interface. It can be imported so you don't need to repeat the component name.

```
import com.example.SomeCompoment.Model
```